docs/developers/write-a-helm-chart.rst

```diff
@@ -5,7 +5,7 @@ Write a Helm chart for an application
 Argo CD manages applications in the Rubin Science Platform through a set of Helm charts.
 Which Helm charts to deploy in a given environment is controlled by the :file:`values.yaml` and :file:`values-{environment}.yaml` files in `/environments <https://github.com/lsst-sqre/phalanx/tree/main/environments/>`__.
```

Comment:
If the intent is to standardize on not using `/` path prefixes, there may be other instances:

```suggestion
Which Helm charts to deploy in a given environment is controlled by the :file:`values.yaml` and :file:`values-{environment}.yaml` files in `environments <https://github.com/lsst-sqre/phalanx/tree/main/environments/>`__.
```